cmd/gopogh/main.go

```diff
@@ -15,7 +15,7 @@ import (
 var Build string
 
 var (
-	dbPath         = flag.String("db_path", "", "path to sql summary output")
+	dbPath         = flag.String("db_path", "", "path to sql summary output. if using cloudsql in the form of 'host=HOST_NAME user=DB_USER dbname=DB_NAME password=DB_PASS'")
```

Comment:
I could be wrong, but based on the new line, it doesn't seem this is a `path to sql summary output`